geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/+package+_geoportal/subscribers.py_tmpl

```diff
@@ -15,8 +17,11 @@ def add_localizer(event):
     localizer = get_localizer(request)
 
     def auto_translate(string):
-        result = localizer.translate(tsf1(string))
-        return localizer.translate(tsf2(string)) if result == string else result
+        for tsf in [tsf1, tsf2, tsf3, tsf4]:
+            result = localizer.translate(tsf(string))
+            if result != string:
+                break
+        return result
```

Comment:
Can't we use deferent `auto_translate` for geoportal and admin?

With for e.g. `if request.path_info.starts_with('/admin/')`?